Plugins/ConsoleControl/ConsoleProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ConsoleControl
+{
+    public interface ConsoleProvider : System.Windows.Forms.IContainerControl
```

Comment:
You are right. I will change this stuff.
Do you want me to remove the DSL commit for this PR?